### PR TITLE
Canvas Options dialog via GtkBuilder

### PR DESCRIPTION
**Pull request by rodolforg**

### CODE REVIEW

**Comment by morevnaproject** [comment]
Tested and it works fine for me!

I have configured Glade to find `synfig-studio/ui/synfig-catalog.xml` file and loaded `canvas_options.ui` file successfully:

screenshot_001

File: synfig-studio/src/gui/resourcehelper.cpp [hunk 1]
```diff
@@ -0,0 +1,104 @@
+/* === S Y N F I G ========================================================= */
+/*!	\file resourcehelper.h
+**	\brief Helper to retrieve the app resource paths, such as icons and plugins
+**
+**	$Id$
+**
+**	\legal
+**	Copyright (c) 2002-2005 Robert B. Quattlebaum Jr., Adrian Bentley
```
Comment (morevnaproject) [inline synfig-studio/src/gui/resourcehelper.cpp]: @rodolforg Please add your copyright here. ^__^

File: synfig-studio/src/gui/resourcehelper.h [hunk 2]
```diff
@@ -0,0 +1,44 @@
+/* === S Y N F I G ========================================================= */
+/*!	\file resourcehelper.h
+**	\brief Helper to retrieve the app resource paths, such as icons and plugins
+**
+**	$Id$
+**
+**	\legal
+**	Copyright (c) 2002-2005 Robert B. Quattlebaum Jr., Adrian Bentley
```
Comment (morevnaproject) [inline synfig-studio/src/gui/resourcehelper.h]: @rodolforg Also, please add your copyright here. ^__^

File: synfig-studio/src/gui/resourcehelper.cpp (hunk 1 above)
Comment (rodolforg) [reply synfig-studio/src/gui/resourcehelper.cpp]: Oh, I really don't care. Git commit logs are enough. lol
Maybe we should do like others projects.
In Godot Engine, for example:
```
/* Copyright (c) 2007-2019 Juan Linietsky, Ariel Manzur.                 */
/* Copyright (c) 2014-2019 Godot Engine contributors (cf. AUTHORS.md)    */
```

Comment (rodolforg) [reply synfig-studio/src/gui/resourcehelper.cpp]: Added it anyway

Comment (rodolforg) [reply synfig-studio/src/gui/resourcehelper.cpp]: Anyway I added it and rebase it (in order to not create a commit just for copyright)

Comment (morevnaproject) [reply synfig-studio/src/gui/resourcehelper.cpp]: > Oh, I really don't care. Git commit logs are enough. lol
> Maybe we should do like others projects.
> In Godot Engine, for example:
> ...

This is actually a great idea! I like it. ^__^

**Comment by morevnaproject** [comment]
Merged via 7ee6fce942e6452f55ef383d4b8c173c607bae27. Thank you!

**Pull request closed without merging**